core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -299,6 +302,34 @@ class PageModel extends Model
 	 */
 	protected $blnDetailsLoaded = false;
 
+	public function __set($strKey, $varValue)
+	{
+		if (\in_array($strKey, array('pageTitle', 'description', 'robots'), true))
+		{
+			trigger_deprecation('contao/core-bundle', '4.12', sprintf('Overriding "%s" is deprecated and will not work in Contao 5.0 anymore. Use the ResponseContext instead.', $strKey));
+
+			$responseContext = System::getContainer()->get(ResponseContextAccessor::class)->getResponseContext();
+
+			if ($responseContext instanceof WebpageContext)
+			{
+				switch ($strKey)
+				{
+					case 'pageTitle':
+						$responseContext->setTitle($varValue);
+						break;
+					case 'description':
+						$responseContext->setDescription($varValue);
+						break;
+					case 'robots':
+						$responseContext->setRobotsMetaTagContent($varValue);
```

Comment:
I feel like a `WebpageContext` could have a generic _Meta tag bag_, similar to a header bag where I can define any meta headers? Could be used to set `og:image` and everything else.

Reply:
Yes, not part of this PR though.